### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Adds missing changelog entries 

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] ~Was the CHANGELOG.md updated to reflect the changes?~
* [ ] ~Was the documentation framework updated to reflect the changes?~
* [ ] ~Have you checked that you haven't introduced any duplicate code?~

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [ ] ~Do all unit tests pass?~
* [ ] ~Do all end-to-end tests pass?~
* [ ] ~Any other testing performed?~
    > ~Tested by {Running the Monkey locally with relevant config/running Island/...}~
* [ ] ~If applicable, add screenshots or log transcripts of the feature working~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,10 +60,15 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Agent hanging if plugins do not shut down. #3557
 - WMI exploiter hanging. #3543
 - Discovered network services are displayed in reports. #3000
+- Windows ransomware README line endings. #3699
+- Services count in scanned servers table in security report. #3708
+- Windows manual run command. #3709
+- Spacing in encrypted files table in ransomware report. #3715
```
Comment (mssalvatore) [inline CHANGELOG.md:66]: I believe the 1st, 3rd, and 4th items in this list are things we broke during the release and subsequently fixed. In other words, I don't think these are bugs in 2.2.0 that are fixed by 2.3.0. As such, they don't belong in the changelog. #2 is a good catch, though.

Feel free to double check me on that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,11 +7,22 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+### Changed
+- Made username and password fields required on login and registration pages. #3795
```
Comment (mssalvatore) [inline CHANGELOG.md:13]: This isn't really meaningful to the user. This is an implementation detail of how we fixed the real issue. The issue we fixed is that error messages were inconsistent. This should be reworded and moved to fixed. It should also reference the issue, not the PR.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,11 +7,22 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
...
-- Ports in Log4Shell exploiter configuration can no longer be floating-point numbers.
-- Ports in MSSQL exploiter configuration can no longer be floating-point numbers.
+- Ports in Hadoop exploiter configuration can no longer be floating-point numbers. #3734
+- Ports in Log4Shell exploiter configuration can no longer be floating-point numbers. #3734
+- Ports in MSSQL exploiter configuration can no longer be floating-point numbers. #3734
+- SMB exploiter configuration bug that wouldn't allow submission. #3849
+- Hadoop exploiter configuration bug that wouldn't allow submission. #3850
 
+### Removed
+- Fingerprinter configuration from UI. #3769, #3826
```
Comment (mssalvatore) [inline CHANGELOG.md:23]: We're planning on adding this back in, though. Right?

**Review by mssalvatore — CHANGES_REQUESTED** [review]
Make sure you're using Issue numbers and not PR numbers. Only use PR numbers if there's no associated issue.

File: CHANGELOG.md (hunk 3 above)
Comment (shreyamalviya) [reply CHANGELOG.md:23]: Yes, I figured we should leave it in for now and remove it when we actually add it back in.

File: CHANGELOG.md (hunk 1 above)
Comment (shreyamalviya) [reply CHANGELOG.md:66]: I think you're right, removed 1st, 3rd, 4th

File: CHANGELOG.md (hunk 3 above)
Comment (mssalvatore) [reply CHANGELOG.md:23]: Ok.

**Pull request merged**